jscomp/bin/whole_compiler.ml

```diff
@@ -81985,9 +81993,10 @@ and ipp_comment : 'a . P.t -> 'a  -> unit = fun   f comment ->
     ]}
 *)
 
-and pp_comment f comment = 
-  if String.length comment > 0 then 
-    P.string f "/* "; P.string f comment ; P.string f " */" 
+and pp_comment f comment =
+  if String.length comment > 0 then
```

Comment:
I do think this is necessary because the out most `/* :: */` is not a part of a `Caml_block` structure, so if we want this behaviour then we have to keep this.